corehq/apps/hqwebapp/models.py

```diff
@@ -214,13 +203,31 @@ def subpage_url_names(self):
         try:
             for name, section in self.sidebar_items:
                 names.extend(subpage['urlname']
-                    for item in section
-                    for subpage in item.get('subpages', []))
+                             for item in section
+                             for subpage in item.get('subpages', []))
```

Comment:
This is an example of a formatting change that is unnecessary. Even though this is an improvement it just makes it harder to see what are real changes and what is just formatting. Generally you should try and leave formatting as is unless you are already editing that piece of code.

If it is really bugging you you can always do a PR with just formatting changes.